argoverse/evaluation/detection_utils.py

```diff
@@ -180,12 +178,24 @@ def iou_aligned_3d(dt_dims: pd.DataFrame, gt_dims: pd.DataFrame) -> np.ndarray:
     return (inter / union).values
 
 
-def normalize_angle(angle: np.ndarray) -> np.ndarray:
-    """Map angle (in radians) from domain [-π, π] to [0, π).
+def wrap(angles: np.ndarray, period: float = np.pi) -> np.ndarray:
```

Comment:
looks good. can we name it `wrap_angle()` or `wrap_0_to_pi()` instead of wrap, just to avoid any confusion with pybind wrappers etc?